### DISCUSSION TITLE
Is this course feasible without Colab?

### DISCUSSION THREAD

**Discussion by MZ-80K** [mrdbourke/pytorch-deep-learning#165] [Q&A] [answered]
I can't use Colab as I don't and won't have a Google account. I can install Pytorch locally within Anaconda but I have no GPU. Is it feasible to follow this course with those constraints?

**Comment by mrdbourke** [comment, marked as answer]
Hi @MZ-80K,

Yes the course is completely doable locally with/without Google Colab.

The models we build in the first ~7 sections are perfectly capable of running without a GPU (though they may take longer to run).

However, models in the later sections will take quite a while without a GPU.

Other options other than Google Colab are:
* Paperspace - https://www.paperspace.com
* AWS Studio Lab - https://studiolab.sagemaker.aws/ 
* Deepnote - https://deepnote.com/home
* Local + setup like you described (however as mentioned above, with no GPU the latter sections may take a considerably long time to run, however, you could always constrain the models/datasets to run in smaller portions)

**Reply by MZ-80K** [reply]
Great, thanks - and for the alternatives too, very useful.
